src/test/fuzz/kitchen_sink.cpp

```diff
@@ -23,4 +25,19 @@ FUZZ_TARGET(kitchen_sink)
     (void)JSONRPCTransactionError(transaction_error);
     (void)RPCErrorFromTransactionError(transaction_error);
     (void)TransactionErrorString(transaction_error);
+
+    (void)StringForFeeEstimateHorizon(fuzzed_data_provider.PickValueInArray({FeeEstimateHorizon::SHORT_HALFLIFE, FeeEstimateHorizon::MED_HALFLIFE, FeeEstimateHorizon::LONG_HALFLIFE}));
```

Comment:
would be nice to introduce an `ALL_FEE_EST_HORIZON` constexpr

Reply:
Fixed!